packages/core/android/build.gradle

```diff
@@ -1,45 +1,101 @@
-apply plugin: 'com.android.library'
 buildscript {
-    repositories {
-        mavenCentral()
-        maven {
-            url 'https://maven.google.com/'
-            name 'Google'
-        }
-    }
-    dependencies {
-        classpath 'com.android.tools.build:gradle:3.2.1'
-        classpath "io.embrace:embrace-swazzler:$emb_android_sdk"
-    }
+  // Buildscript is evaluated before everything else so we can't use getExtOrDefault
+  def kotlin_version = rootProject.ext.has("kotlinVersion") ? rootProject.ext.get("kotlinVersion") : project.properties["RNEmbraceCore_kotlinVersion"]
+
+  repositories {
+    google()
+    mavenCentral()
+  }
+
+  dependencies {
+    classpath "com.android.tools.build:gradle:8.6.0"
+    classpath "io.embrace:embrace-swazzler:$emb_android_sdk"
```

Comment:
ah I see `emb_android_sdk` is still being read here, if possible we should have it grabbed from package.json as well, if not may need to restore part of `packages/core/update_android_version.ts` since it was previously keeping the value for it in sync

Reply:
I'm in favor of getting rid of `emb_android_sdk` but this change may impact in other places as well like the install script + related tests. I guess I would vote for this to get `packages/core/update_android_version.ts` back and clean it up in a next iteration